en/identity-server/next/docs/guides/authorization/api-authorization/attribute-based-access-control.md

```diff
@@ -1,6 +1,197 @@
+2. Click **Add User** button and select **Single User** option, provide the relevant details and click 
+   **Save & Continue**.
+3. Go to the **User Management** and click **Roles**.
+4. Click **New Role**, add the Basic Details, Permission Selection and click **Finish**.
+5. Go to the **Users** tab of the created role and assign the created user to the role.
+
+    !!!note
+        For more information on User Management, see [here](../../users/index.md)
+
+The next step is to configure the XACML policy to validate the XACML scope during OAuth
+token issuance.
+
+### Set up the policy
+
+Follow the instructions given below to publish a policy using a XACML policy
+template that is available by default with WSO2 Identity Server.
+
+1. Log in to the Management Console via <https://localhost:9443/carbon>.
+2. In the **Main** tab of the Management Console, navigate to
+   **PAP** \> **Policy Administration** under the **Entitlement** menu.
+3. Select the `scope_based_token_issuance_policy_template`
+   policy, and click **Edit** to view the selected policy in the policy
+   editor.
+   
+
+    !!! info        
+        XACML template policies provide a pre-configured template with place
+        holders to customize the policy depending on your requirement.
+
+
+4. Edit the policy to customize it depending on your requirement. You
+   can change the values of attributes and rules.
+5. Click **Save Policy** to save the changes. You can see the policy
+   that you created on the policy list (the original policy template
+   remains unchanged).
+6. Click the link **Publish to My PDP** corresponding to the new
+   policy.
+7. On the UI that appears, leave the default values as they are and
+   click **Publish**.
+
+   To ensure that the policy has been published successfully, click on
+   **Policy View** under the **Entitlement \> PDP** section on the
+   **Main** tab of the Management Console, and check if the created
+   policy is listed.
+
+Now that you have created the policy and enforced it using the policy
+template, you can test the policy to evaluate whether XACML scope is
+validated at the time of OAuth token validation.
+
+### Try it out
+
+Follow the steps given below to try out the policy using the above created application.
+
+1. On the {{ product_name }} Console, go to **Applications** and open the created application.
+2. Go to the **Protocols** tab of the application and enable the **Password** grant type.
+3. Execute the following cURL command to get an access token using the password grant type.
+
+```java
+curl --location 'https://localhost:9443/t/<TENANT_DOMAIN>/oauth2/token' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--header 'Authorization: Basic <BASE64_ENCODED(CLIENT_ID:CLIENT_SECRET)>' \
+--data-urlencode 'grant_type=password' \
+--data-urlencode 'username=<USER_NAME>' \
+--data-urlencode 'password=<USER_PASSWORD>' \
+--data-urlencode 'scope=openid SCOPE_1'
```

Comment:
```suggestion
--data-urlencode 'scope=openid <SCOPE_1> <SCOPE_2>'
```

Reply:
SCOPE_1 is the actual scope name that we need to validate from the XACML policy